src/bitcoin/poller/looper.rs

```diff
@@ -208,13 +208,11 @@ fn new_tip(bit: &impl BitcoinInterface, current_tip: &BlockChainTip) -> TipUpdat
 }
 
 fn updates(
+    db_conn: &mut Box<dyn DatabaseConnection>,
```

Comment:
Commit a8ca8500a4c60d2e135508d7efcbb1f6ef784f2e "poller: hold the same database lock across one update round" is confused. This is not a lock, it's a connection.